7/dotnetcore-2.2.110/Dockerfile

```diff
@@ -35,15 +35,19 @@ RUN curl -SL --output dotnet.tar.gz https://dotnetcli.blob.core.windows.net/dotn
 
 ENV DETECT_DOTNET_PATH /usr/share/dotnet/dotnet
 
-# Download detect script
-RUN mkdir -p /app
-RUN curl -o /app/detect.sh https://detect.synopsys.com/detect.sh
+# Download detect 6 script
+RUN mkdir -p /app \
+    && curl -o /app/detect6.sh https://detect.synopsys.com/detect.sh \
+    && chmod +x /app/detect6.sh \
```

Comment:
Is this intended to allow running version 6 on request?

Reply:
Blackduck offers 6 as default for running detect. I don't like that... I want the default to be the most advanced version of Blackduck, but if you do need 6, I will offer it in this in the image.